.github/workflows/release-winget.yml

```diff
@@ -0,0 +1,30 @@
+name: Submit Winget Package to Windows Package Manager Community Repository
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: "Specific tag name"
+        required: false
+        type: string
+env:
+  CI: true
+jobs:
+  winget-publish:
+    name: Publish winget package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set winget version env
+        env:
+          TAG_NAME: ${{ inputs.tag_name || github.event.release.tag_name }}
+        run: echo "WINGET_TAG_NAME=$(echo ${TAG_NAME#v})" >> $GITHUB_ENV
+      - name: Submit package to Windows Package Manager Community Repository
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: Triplex.Triplex
+          installers-regex: '^Triplex-Setup-[\d\.]+\.exe$'
+          version: ${{ env.WINGET_TAG_NAME }}
+          release-tag: ${{ inputs.tag_name || github.event.release.tag_name }}
+          token: ${{ secrets.WINGET_ACC_TOKEN }}
+          # fork-user: triplex-bot
```

Comment:
Does this need to be uncommented?

Reply:
If triplex-bot is the one doing the release, then yes! I'm editing the PR quickly

Reply:
So it does the draft releases but the released ones are done manually

Reply:
I do think both manual and automatic (on-release) triggering of the action would push an update of the winget package the winget-phgs. Maybe the [winget-releaser](https://github.com/vedantmgoyal9/winget-releaser) action can be setup to filter-out draft vs standard releases - but looking at the doc, this spawns: 

> The action will only work when the release is published (not a draft), because the release assets (binaries) aren't available publicly until the release is published.

Reply:
Great let's send it